Source/Diagnostics/FlushFormats/FlushFormatPlotfile.cpp

```diff
@@ -367,7 +367,7 @@ FlushFormatPlotfile::WriteParticles(const std::string& dir,
             }, true);
         } else {
             PinnedMemoryParticleContainer* pinned_pc = particle_diags[i].getPinnedParticleContainer();
-            tmp.copyParticles(*pinned_pc);
+            tmp.copyParticles(*pinned_pc, true);
```

Comment:
Does the same change need to go into `Source/Diagnostics/WarpXOpenPMD.cpp`? :)

Reply:
thanks for checking this! yes and commit 8dd66f4 adds this